### PR TITLE
{bio}[foss/2021b] msprime v1.2.0 w/ Python 3.9.6

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by akesandgren** [comment]
@boegelbot Please test @ jsc-zen2

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
b-an02.hpc2n.umu.se - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 3.8.10
See https://gist.github.com/7c8c2669b6aa4f51c627010f40b0280d for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
b-an02.hpc2n.umu.se - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 3.8.10
See https://gist.github.com/fa81b95b7c38aeffef6428646226bd3c for a full test report.

File: easybuild/easyconfigs/m/msprime/msprime-1.2.0-foss-2021b.eb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    }),
+    (name, version, {
+        'checksums': ['60025ad9fd30d827a72ae6e760b6cff07a1d0e169b2c1da1024cb0fc23e4a7aa'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': ['bin/msp', 'bin/mspms', 'bin/tskit', 'bin/demes'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
```
Comment (verdurin) [inline easybuild/easyconfigs/m/msprime/msprime-1.2.0-foss-2021b.eb:49]: @akesandgren don't suppose you feel like adding a `sanity_check_command`?

Comment (akesandgren) [reply easybuild/easyconfigs/m/msprime/msprime-1.2.0-foss-2021b.eb:49]: can do...

Comment (akesandgren) [reply easybuild/easyconfigs/m/msprime/msprime-1.2.0-foss-2021b.eb:49]: And done.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
b-an02.hpc2n.umu.se - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 3.8.10
See https://gist.github.com/5074d664d249d7ee27df57d7e2c4e574 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/effaae4a04b0b16b35538b7cbd5fd50d for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ jsc-zen2

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @akesandgren!

**Pull request merged**